### DISCUSSION TITLE
When will the new Nodejs 16.x LTS  version release?

### DISCUSSION THREAD

**Discussion by timescodex** [nodejs/help#4920] [Q&A]
There is high CVE for minimatch https://github.com/nodejs/node/issues/42510 and need to know when will the new 16.x LTS version release, anyone can help. Thanks.

**Comment by Trott** [comment]
Run `npm install -g npm` to resolve it.

Or just ignore it. This does not affect the `node` binary and is only going to affect `npm` if you install untrusted packages, in which case you're already vulnerable to a lot worse than this.

**Comment by Trott** [comment]
It is hoped that the release will go out Thursday (so in the next 48 hours) but it is dependent on resolving an issue that is delaying the release. https://github.com/nodejs/Release/issues/658#issuecomment-1095752254
